R/theme-defaults.r

```diff
@@ -134,7 +134,7 @@ theme_grey <- function(base_size = 11, base_family = "") {
     panel.background =   element_rect(fill = "grey92", colour = NA),
     panel.border =       element_blank(),
     panel.grid.major =   element_line(colour = "white"),
-    panel.grid.minor =   element_line(colour = "white", size = 0.25),
+    panel.grid.minor =   element_line(colour = "white", size = rel(0.25)),
```

Comment:
Ooops, should've refreshed before commenting. Glad the tests picked this up! (Although I'm not sure what went wrong on travis)